OneCommunityGlobal/HGNRest
# Yili - Gives the user permission to Reset and/or Change the password of any user

# Description

![Gives the user permission to Reset and:or Change the password of any user but Owner:Admin classes](https://github.com/user-attachments/assets/fec85cf7-eed8-40d2-bb6e-d6cbba59eb37)

## Related PRS
To test this backend PR you need to checkout the frontend [PR #2860 ](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2860)

## How to test:
1. check into current branch
2. do npm install and ... to run this PR locally
3. follow the steps on the frontend [PR #2860 ](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2860)
